### ISSUE TITLE
commeted code, should clear after new line

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#751]
the following should report commented code, clear after newline

```abap
* todo
*    DATA(lv_abap) = mo_cut->build_meta( ls_generate ).
*
*    cl_abap_unit_assert=>assert_not_initial( lv_abap ).
*    cl_abap_unit_assert=>assert_char_cp(
*      act = lv_abap
*      exp = '*MOO*' ).
```

**Comment by g-back** [comment]
similarly, not reported:
```abap
" DATA(lv_abap) = mo_cut->build_meta( ls_generate ).
" cl_abap_unit_assert=>assert_not_initial( lv_abap ).
" cl_abap_unit_assert=>assert_char_cp(
"    act = lv_abap
"    exp = '*MOO*' ).
```
only comment blocks with * are currently checked
